plugins/syntax/float_syntax.ml

```diff
@@ -20,8 +20,58 @@ let make_path dir id = Libnames.make_path (make_dir dir) (Id.of_string id)
 
 (*** Parsing for float in digital notation ***)
 
+let warn_inexact_float =
+  CWarnings.create ~name:"inexact-float" ~category:"parsing"
+    ~default:CWarnings.AsError
+    (fun (sn, f) ->
+      Pp.strbrk
+        (Printf.sprintf
+           "The constant %s is not a binary64 floating-point value. \
+            To turn this as a warning and use a closest value, which will \
+            unambiguously be printed %s, use option \"-w inexact-float\"."
```

Comment:
I think this should say something like "disable warning inexact-float" or otherwise avoid implying that going through the command line is necessary.

Reply:
I agree. Would suggesting the vernacular `Set Warnings "inexact-float"` work? Something like "The constant %s is not a binary64 floating-point value. One can Set Warnings "inexact-float" to turn this into a warning and use a closest value, which will unambiguously be printed %s."